lib/oracle-store.js

```diff
@@ -355,50 +423,52 @@ function fixquery(qent,q) {
   if( !q.native$ ) {
     for( var qp in q ) {
       if( !qp.match(/\$$/) ) {
-        qq[qp] = q[qp]
+        qq[qp] = q[qp];
       }
     }
   }
   else {
-    qq = _.isArray(q.native$) ? q.native$[0] : q.native$
+    qq = _.isArray(q.native$) ? q.native$[0] : q.native$;
   }
 
-  return qq
+  return qq;
 }
 
 
 function metaquery(qent,q) {
-  var mq = {}
+  var mq = {};
 
   if( !q.native$ ) {
 
     if( q.sort$ ) {
-      for( var sf in q.sort$ ) break;
-      var sd = q.sort$[sf] < 0 ? 'DESC' : 'ASC'
-      mq.sort = [[sf,sd]]
+      for( var sf in q.sort$ ) {
+          break;
+      }
+      var sd = q.sort$[sf] < 0 ? 'DESC' : 'ASC';
+      mq.sort = [[sf,sd]];
     }
 
     if( q.limit$ ) {
-      mq.limit = q.limit$
+      mq.limit = q.limit$;
     }
 
     if( q.skip$ ) {
-      mq.skip = q.skip$
+      mq.skip = q.skip$;
     }
 
     if( q.fields$ ) {
-      mq.fields = q.fields$
+      mq.fields = q.fields$;
     }
   }
   else {
-    mq = _.isArray(q.native$) ? q.native$[1] : mq
+    mq = _.isArray(q.native$) ? q.native$[1] : mq;
   }
 
-  return mq
+  return mq;
 }
 
 
-function fixid(ent) {
+var fixid = function (ent) {
```

Comment:
I'm not sure about this (from my tests the former is faster)... but the difference is negligible.